recipe/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set version = "3.4.1.dev3" %}
+{% set name = "whylogs-sketching" %}
+{% set sha256 = "40b90eb9d5e4cbbfa63f6a1f3a3332b72258d270044b79030dc5d720fddd9499" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [py<37 or (win and vc<16)]
+  script: python -m pip install --no-deps --ignore-installed --no-build-isolation .
```

```suggestion
  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed --no-build-isolation . -v`
```
I think it's preferable to use `{{ PYTHON }}` instead of `python` to make sure that the right python executable is used. `-v` will make it so that we have logs from the compiler.